scalableminds/webknossos
# Histogram decimal places

This PR allows for more decimal places in the histogram ranges. Furthermore the slider's tooltip displays an appropriate amount of decimal places as well.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing with at least one color layer
- edit a histogram's range
- it should be possible to enter several decimal places
- the slider's tooltip should display up to three more decimal places than the range min/max values

### Issues:
- fixes #4753
------
